components/script/dom/dedicatedworkerglobalscope.rs

```diff
@@ -173,7 +173,7 @@ impl DedicatedWorkerGlobalScope {
 
             let roots = RootCollection::new();
             let _stack_roots_tls = StackRootTLS::new(&roots);
-            let (url, source) = match load_whole_resource(LoadContext::Script,
+            let (metadata, bytes) = match load_whole_resource(LoadContext::Script,
```

Comment:
Nit: realign the other arguments to `load_whole_resource` so they line up again.